database/gorm/query.go

```diff
@@ -455,6 +455,25 @@ func (r *QueryImpl) OrderByDesc(column string) ormcontract.Query {
 	return r.Order(fmt.Sprintf("%s DESC", column))
 }
 
+func (r *QueryImpl) InRandomOrder() ormcontract.Query {
+	order := ""
+	switch r.Driver() {
+	case "mysql":
```

Comment:
S: Use `orm.DriverMysql.String()` instead.

Reply:
when I use Postgresql database `r.Driver()` returns "postgres" but `ormcontract.DriverPostgresql` returns "postgresql" and it causes a problem

Reply:
what should I do in this case?
I can define a default value for `order` variable. 
like this: `order := "RAND()"`

Reply:
We can define a new constant: `ormcontract.DriverPostgres`, we want to use `postgres` in the future.